python_modules/dagma/dagma/utils.py

```diff
@@ -14,6 +17,25 @@ def mkdir_p(path):
             raise
 
 
+@contextlib.contextmanager
```

Comment:
we should maybe reconcile this with get_temp_file_names and get_temp_file_name in dagster.utils.test